vtex-apps/google-import
# (CATIMPORT-31) Fix to setting category parent id

When using the google import app, the categories were not created correctly. Instead of creating a parent/child relationship, it created 2 different parent categories
Pricing Error:  Price: [NotFound] The route [PUT] /pricing/prices/ does not exist.

Fixed method for iterating category levels
For pricing: When a new product is created, set product id from response.

To Test
Create a new child category using import to a V2 catalog.  Confirm parent/child relationship
Set price values using import.  Confirm prices are set.
